tests/bluetooth/tester/src/mesh.c

```diff
@@ -465,6 +492,16 @@ static void provision_node(uint8_t *data, uint16_t len)
 	flags = cmd->flags;
 	iv_index = sys_le32_to_cpu(cmd->iv_index);
 	net_key_idx = sys_le16_to_cpu(cmd->net_key_idx);
+
+	if (len > sizeof(*cmd)) {
```

Comment:
That doesn’t seem sufficient. Don’t you want if (len >= sizeof(*cmd) + sizeof(pub_key))?

Reply:
@jhedberg - I would prefer not to change the ordering of any backports for v2.7 or otherwise introduce any inconsistencies, so if there needs to be a rework of this code, please open a new issue and set it to medium priority.